packages/fxa-content-server/app/scripts/views/mixins/cached-credentials-mixin.js

```diff
@@ -93,6 +93,12 @@ export default {
         throw err;
       }
     });
+
+    // When using a cached credential, the auth-server routes do not get hit,
+    // This event will cause the content-server to emit the complete event.
+    this.logEvent('cached.signin.success');
```

Comment:
Fwiw, I think this is wrong because we'll still emit the success event if the call to `this.signIn` fails. We only want to emit it when the call succeeds.

Reply:
Ref: #2358

Reply:
Also, we should have added a test for this.